### PR TITLE
Test: Added test for UserInfoDesktopView

### PR DESCRIPTION
**Pull request by ManyRios**
## Describe your changes

Added test units for src/people/userInfo/UserInfoDesktopView.tsx

image

## Issue ticket number and link
https://github.com/stakwork/sphinx-tribes-frontend/issues/314#issuecomment-1968335483

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

File: src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    });
+  });
+
+  it('Test that description/bio for a user is visible if provided', async () => {
+    render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+    const description = screen.getByTestId('user-description');
```
Comment (elraphty) [inline src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx:53]: @ManyRios assert that the description text matches the mock user description

File: src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  it('Test that price to connect, pubkey , copy button, email if provided, and github account if provided are visible', async () => {
+    render(
+      <MockStoreEnvironment hooks={[MOCK_ENVIRONMENT_HOOKS.SELF_PROFILE_STORE]}>
+        <UserInfoDesktopView setShowSupport={() => null} />
+      </MockStoreEnvironment>
+    );
+
+    waitFor(() => {
+      const price = screen.getByText('Price to Connect:');
+      expect(price).toBeInTheDocument();
```
Comment (elraphty) [inline src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx:105]: @ManyRios Assert that all provided data matches the data on the mock user.

File: src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+  it('Test that user skills are visible and rendered if provided', () => {
+    render(
+      <MockStoreEnvironment>
+        <UserInfoDesktopView setShowSupport={() => null} />
+      </MockStoreEnvironment>
+    );
+
+    waitFor(() => {
+      const skills = screen.getByTestId('user-skills');
```
Comment (elraphty) [inline src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx:131]: Assert that the user skills match that of the mock user.

File: src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        <UserInfoDesktopView setShowSupport={setShowSupport} />
+      </MockStoreEnvironment>
+    );
+
+    expect(user.getByText('Connect')).toBeInTheDocument();
+
+    await act(async () => {
+      user.getByText('Connect').click();
+      await waitFor(() => {
+        const modal = user.getByTestId('connect-modal');
```
Comment (kevkevinpal) [inline src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx:70]: lets assert that the text on the modal appears

`Discuss this bounty with` `Connect with Sphinx` ect
Screenshot 2024-02-29 at 3 52 25 PM

File: src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+      </MockStoreEnvironment>
+    );
+    await act(async () => {
+      const sendTip = screen.getByText('Send Tip');
+      expect(sendTip).toBeInTheDocument();
+
+      waitFor(async () => {
+        fireEvent.click(sendTip);
+        const modal = await screen.getByTestId('modal_support');
+        expect(modal).toBeInTheDocument();
```
Comment (kevkevinpal) [inline src/people/userInfo/__test__/UserInfoDesktopView.spec.tsx:90]: We might want to assert some of the text in this modal aswell

**Review by kevkevinpal — COMMENTED** [review]
I think this PR is looking good

nice job

I think we can just assert more text in the modals and @elraphty's suggestions

**Comment by ManyRios** [comment]
@elraphty @kevkevinpal Done!!!!!!!!!!!!!!!!

**Pull request merged**